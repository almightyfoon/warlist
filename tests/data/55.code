c85b1b_-0qg1_Y4OgZ4M4M_Y5l4H4H_Yc95j5j4S5s4U4Wgu__4w_Y5K4B4Q4M4z_Y4z4H4H5AfrfF_Y5e_Y5e4Z4_gs