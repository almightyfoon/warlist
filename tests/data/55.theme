{"allowed":[12,15,46,47,75,158,163,190,207,215,228,233,234,235,236,239,276,320,323,334,335,336,337,339,340,348,350,385,386,387,389,390,391,392,393,394,395,396,397,398,401,403,405,406,407,409,410,411,413,414,416,420,422,423,427,432,433,434,445,452,455,456,457,458,460,461,462,466,472,474,476,477,478,479,481,482,483,484,485,486,487,488,499,500,501,503,504,505,513,514,515,517,518,519,520,522,523,525,526,527,581,654,655,676,685,688,696,702,703,704,726,785,822,2048,2049,2053,2066,3023,3059,3060,3061,3062,3128,3129,3130,3143,3155,3176,3177,3178,3180,4015,4029,4040,6003,6030,6060,6064,6066,6140,6160,6177,6187,6250,6261,6262,6263,6264,6265,6266,6294,6302,6329,6393,6397,6403,6404,6405,6406,6407,6408,6409,6410,6411,6412,6414,6420],"required":[432,433,434,445,461,462,476,4029,6066,6177,6250,6262,6263,6264,6265,6266],"free":[429,436,447,462,476,477,657,703,4029,6066,6262,6263,6265,6266]}