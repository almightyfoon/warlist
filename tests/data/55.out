https://conflictchamber.com/#c85b1b_-0qg1_YmDmW4M4M_YmJ4H4H_YmV5j5j4S5s4U4Wgu__4w_YmK4B4Q4M4z_Y4z4H4H5AfrfF_Y5e_Y5emF4_gs

Circle Army - 75 / 75 points
[Theme] The Wild Hunt

[Tanith 1] Tanith the Feral Song [+31]
 - Feral Warpwolf [16]
 - Loki [19]
 - Pureblood Warpwolf [17]
 - Pureblood Warpwolf [17]
 - Warpwolf Stalker [17]
 - Woldwyrd [9]
 - Woldwyrd [9]
Gallows Grove [2]
War Wolf [2]
War Wolf [2]
Reeves of Orboros (max) [16]
 - Reeve of Orboros Chieftain & Standard [0(4)]
Wolves of Orboros (max) [11]
 - Wolves of Orboros Chieftain & Standard [4]
Fuel Cache (Objective) [0]

Circle Army - 74 / 75 points

[Mohsar 1] Mohsar the Desertwalker [+27]
 - Ghetorix [19]
 - Gorax Rager [7]
 - Megalith [20]
 - Pureblood Warpwolf [17]
 - Wild Argus [7]
 - Wild Argus [7]
 - Woldwyrd [9]
 - Woldwyrd [9]
 - Druid Wilder [4]
Bloodweaver Night Witch [4]
Swamp Gobber Chef [1]
Sentry Stone & Mannikins [5]
Sentry Stone & Mannikins [5]
Tharn Bloodtrackers (max) [15]
Tharn Bloodweavers [8]
Bunker (Objective) [0]
